Masquerade4997Library/src/main/java/Library4997/MasqSensors/MasqAdafruitIMU.java

```diff
@@ -24,6 +24,7 @@
     double zeroPos = 0;
     public MasqAdafruitIMU(String name, HardwareMap hardwareMap) {
```

Comment:
If you're initialization happens in the constructor why does the initalizeIMU method exist?